grc/core/platform.py

```diff
@@ -52,9 +52,6 @@ def __init__(self, *args, **kwargs):
         self._block_categories = {}
         self._auto_hier_block_generate_chain = set()
 
-        if not yaml.__with_libyaml__:
-            logger.warning("Slow YAML loading (libyaml not available)")
```

Comment:
Are you sure we want to remove this warning?

Reply:
The warning hasn't been printed with slow loading in the past, and if it is printed, there's nothing the average user can do about it, so it just adds noise.